WordPress/Classes/Services/BlogService.m

```diff
@@ -744,6 +734,27 @@ - (NSTimeZone *)timeZoneForBlog:(Blog *)blog
     return timeZone;
 }
 
+- (void)updateBlog:(Blog *)blog settingsWithRemoteSettings:(RemoteBlogSettings *)remoteSettings
+{
+    blog.blogName = remoteSettings.name;
+    blog.blogTagline = remoteSettings.desc;
+    if (remoteSettings.defaultCategory) {
```

Comment:
Right, but assigning a nil value to `defaultCategoryID` / `defaultPostFormat` / `siteVisibility` shouldn't hurt (right)?